_rules/role-attribute-has-valid-value-674b10.md

```diff
@@ -11,10 +11,14 @@ accessibility_requirements:
     passed: further testing needed
     inapplicable: further testing needed
 input_aspects:
-  - DOM Tree
-  - CSS Styling
+ - DOM Tree
+ - CSS Styling
 authors:
-  - Jey Nandakumar
+ - Jey Nandakumar
+htmlHintIgnore:
+ # https://www.npmjs.com/package/htmlhint 
+ # (used with `npm test` to ensure validity of code snippets)
+ - 'src-not-empty'
```

Comment:
IMO we shouldn't have empty src's. We should just put some image there that we have the appropriate alt attribute for. Otherwise the test is incorrect.